src/vcf/validator.cpp

```diff
@@ -20,6 +20,12 @@ namespace ebi
 {
   namespace vcf
   {
+    bool validate(const std::vector<char> &firstLine,
```

Comment:
I'm ok with the static option, but I still see easier to understand that something has internal linkage having the declaration in the cpp. 

I see the need of this `static` usage when you only have a hpp file and don't have a cpp file, but here we do have both.

In general, facing two equivalent options I'd prefer the simpler one that doesn't confuse even the c++'s standard commitee. Even more with static, that can mean lots of different things depending on where it is.